src/platform/visionfive2.rs

```diff
@@ -79,8 +79,8 @@ impl Platform for VisionFive2Platform {
     }
 
     fn get_miralis_memory_start_and_size() -> (usize, usize) {
-        let size = FIRMWARE_START_ADDR - MIRALIS_START_ADDR;
-        (MIRALIS_START_ADDR, size)
+        //let size = FIRMWARE_START_ADDR - MIRALIS_START_ADDR;
+        (MIRALIS_START_ADDR, 0x200000) // Temporary workaround, suggestions appreciated
```

Comment:
I remember adding a comment here but I can't find it anymore, so I probably discarded it by mistake...

There are symbols for the start of Miralis (`_start_address`) and for the start of the stack (`_stack_start`) The size of Miralis is at `_stack_start - _start_address + PLATFORM_STACK_SIZE * PLATFORM_NB_HARTS`.

You will need to round the address to a power of 2 however, because we use it in the PMP with a NAPOT encoding (which requires sizes that are powers of two).

Reply:
I believe that's on another branch, #126